src/sidebar/store/create-store.ts

```diff
@@ -158,14 +158,21 @@ export function createStore<
     thunk,
   ];
 
-  const enhancer = redux.applyMiddleware(...defaultMiddleware, ...middleware);
+  // In debug builds, allow using redux dev tools
+  const isDebug = process.env.NODE_ENV !== 'production';
+  const composeEnhancers =
+    (isDebug && (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) ||
+    redux.compose;
```

Comment:
How much of a risk is there of problems due to us using a different version of Redux than the devtools extension that is installed? I see we are only using the extension's `compose` function in debug builds, which is good.

Reply:
I guess the risk is low. In the worst case scenario it could break in non-prod envs, but it could also hide some actual issue that we end up not being aware of until deploying to prod, so let me check exactly how this is handled by the extension.

Reply:
@robertknight I haven't found anything specific to this. The extension currently depends on `"redux": "^4.2.1"`, which overlaps with what we depend on. However, redux 5 is on its way, and I don't know how soon will the extension be updated.

I have also checked that the implementation of `__REDUX_DEVTOOLS_EXTENSION_COMPOSE__` in the extension does not directly use any redux internal function. Everything is custom implementation. However, I'm almost sure it works on assumptions that might change for redux 5.

So I'm not sure how to proceed here, to be honest.